### DISCUSSION TITLE
jarsigner: unable to open jar file error

### DISCUSSION THREAD

**Discussion by adriantzn** [adempiere/adempiere#4163] [Q&A]
Im having this error "jarsigner: unable to open jar file: C:\Adempiere\lib\AdempiereSLib.jar" on setup

**Comment by cayenet** [comment]
Anyone?? I have the same problem

**Reply by yamelsenih** [reply]
Hello @cayenet can you use a issue instead? Note that we have a issue form with main data like java versión, SO anothers
